5-AccessControl/1-call-api-roles/AppCreationScripts/sample.json

```diff
@@ -1,104 +1,102 @@
 {
-  "Sample": {
-    "Title": "React single-page application calling a protected web API using App Roles to implement Role-Based Access Control",
-    "Level": 300,
-    "Client": "React SPA",
-    "Service": "Node.js web API",
-    "RepositoryUrl": "ms-identity-javascript-react-tutorial",
-    "Endpoint": "AAD v2.0"
-  },
-  "AADApps": [
-    {
-      "Id": "service",
-      "Name": "msal-node-api",
-      "Kind": "WebApi",
-      "Audience": "AzureADMyOrg",
-      "HomePage": "https://localhost:5000/api",
-      "AppRoles": [
-        {
-          "Types": [ "User" ],
-          "Name": "TaskAdmin",
-          "Description": "Admins can read any user's todo list"
-        },
-        {
-          "Types": [ "User" ],
-          "Name": "TaskUser",
-          "Description": "Users can read and modify their todo lists"
-        }
-      ],
-      "ManualSteps": [
-        { "Comment": "Navigate to the app manifest and set 'accessTokenAcceptedVersion' to 2" },
-        { "Comment": "To receive the 'roles' claim with the name of the app roles this user is assigned to, make sure that the user accounts you plan to sign-in to this app is assigned to the app roles of this service instance. The guide, https://aka.ms/userassignmentrequired provides step by step instructions." },
-        { "Comment": "Or you can run the ..\\CreateUsersAndAssignRoles.ps1 command to automatically create a number of users, and assign these users to the app roles of this app." }
-      ]
+    "Sample": {
+        "Title": "React single-page application calling a protected web API using App Roles to implement Role-Based Access Control",
```

Comment:
>React single-page application calling a protected web API using App Roles to implement Role-Based Access Control [](http://example.com/codeflow?start=18&length=112)

Add authorization using app roles to a React single-page app that signs-in users and calls a protected Node.js Web Api